docs/source/snowpark/functions.rst

```diff
@@ -348,6 +363,7 @@ Functions
     to_date
     to_decimal
     to_double
+    to_file
```

Comment:
I don't see  them
  - `ai_filter`
  - `ai_agg`
  - `summarize_agg`

Reply:
They were added in previous PR